subsys/bluetooth/controller/ll_sw/ll.c

```diff
@@ -175,7 +175,10 @@ int ll_init(struct k_sem *sem_rx)
 		return -ENOMEM;
 	}
 
+#if defined(CONFIG_BT_CTLR_FILTER)
```

Comment:
Possible to use `IS_ENABLED()` here?

Reply:
I included this change commit from #15780 just to get CI tests to work (as I added new conf files to testcases.yaml). I will remove the commit when #15780 gets merged first.

I kept the #15780 with the commit seperate in order to have backport label on that commit.